server/log-collector/pkg/services/logcollector/server.go

```diff
@@ -730,9 +757,11 @@ func (s *Server) startLogStreaming(ctx context.Context,
 
 // streamPodLogs streams logs from a pod to a file
 func (s *Server) streamPodLogs(ctx context.Context,
-	runUID string,
+	runUID,
```

Comment:
Accidentally removed?
```suggestion
	runUID string,
```

Reply:
Nope, in Go if several consecutive arguments have the same type, you can put it only on the last one.